lib/versioning/node/schedule.ts

```diff
@@ -14,3 +15,21 @@ export const nodeSchedule: NodeJsData = JSON.parse(
   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
   dataFiles.get('data/node-js-schedule.json')!
 );
+
+export function findScheduleForCodename(
+  codename: string
+): ({ version: string } & NodeJsSchedule) | null {
+  for (const version of Object.keys(nodeSchedule)) {
+    const schedule = nodeSchedule[version];
+    if (schedule.codename?.toLowerCase() === codename.toLowerCase()) {
```

Comment:
Use `toUpperCase` for comparison, as some chars have no lower case representation.